### PR TITLE
3.0.x - bake seed

### PR DESCRIPTION
**Pull request by saeideng**

### CODE REVIEW

**Comment by saeideng** [comment]
test passes locally
```
phpunit tests/TestCase/Command/BakeSeedCommandTest
```
if this test passes in Travis I will convert remains shells in next PR

File: tests/TestCase/TestCase.php [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+     * @param string $message The error message.
+     * @return void
+     */
+    protected function assertFileNotContains($expected, $path, $message = '')
+    {
+        $this->assertFileExists($path, 'Cannot test contents, file does not exist.');
+
+        $contents = file_get_contents($path);
+        $this->assertStringNotContainsString($expected, $contents, $message);
+    }
```
Comment (markstory) [inline tests/TestCase/TestCase.php]: I wonder if we should move these into the ConsoleIntegrationTestTrait?

Comment (saeideng) [reply tests/TestCase/TestCase.php]: yes, it is helpful

**Comment by saeideng** [comment]
I restarted Travis no error now on `bakeSeed` tests
we can merge now , remains error most related to `shell` that will convert to command  soon

File: tests/TestCase/TestCase.php (hunk 1 above)
Comment (saeideng) [reply tests/TestCase/TestCase.php]: I will remove this after having this method on core

**Pull request merged**